docs/Makefile

```diff
@@ -75,4 +75,10 @@ multiversion: setup
 
```

Comment:
@lauranovich After the line 12 (Makefile), please add the following new line:

```
TESTSPHINXOPTS  = $(ALLSPHINXOPTS) -W --keep-going
````

I'm updating the upgrade guide to reflect this change.